torch/ao/quantization/quantization_types.py

```diff
@@ -11,8 +11,11 @@
 # TODO(future PR): improve this.
 QuantizerCls = Any
 
+ExampleInputs = Tuple[Any, ...]
```

Comment:
Is it possible to use more specific types than Any? Is `torch.Tensor` not general enough?

Reply:
yeah this can be (nested) List, Dict, Tuple as well